cluster-autoscaler/fix_gopath.sh

```diff
@@ -11,7 +11,8 @@ for item in $to_fix; do
    mkdir "${GOPATH}/src/k8s.io/${item}"
    cd "${GOPATH}/src/k8s.io/${item}"
    git init
-   cp -R "${GOPATH}/src/k8s.io/kubernetes/staging/src/k8s.io/${item}/*" ./
+   # shellcheck disable=SC2086
+   cp -R ${GOPATH}/src/k8s.io/kubernetes/staging/src/k8s.io/${item}/* ./
```

Comment:
There are other places below this that need the same treatment.

Reply:
All fixed :)